ui/src/MobileClientBuildList.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import { ListGroup, ListGroupItem, Icon, Row, Col } from 'patternfly-react';
+import './styles/MobileClientBuild.css';
+
+export class MobileClientBuildList extends Component {
+    render = () => {
+        const mobileClientBuilds = Object.values(this.props.mobileClientBuilds || {});
+        return ( 
+        <div className="text-left">
+            <h6 className="list-group-heading">
+                MOBILE BUILDS
+            </h6>
+            <ListGroup className="list-group-build">
+                    {mobileClientBuilds.map(
+                        ( mobileClientBuild, index ) => (
+                        <ListGroupItem key={index}>
+                            <Row>
+                                <Col md={6} className="text-left list-group-build-item">
+                                        <a href={mobileClientBuild.metadata.selfLink}>{mobileClientBuild.status.config.name} </a>
+                                </Col>
+                                <Col md={6} className="list-group-build-item-description">
+                                <p className="build-info"> 
+                                        {(mobileClientBuild.status.phase === "Running") ?
+                                                <Icon type="fa" name="refresh" className="icon"/> : (mobileClientBuild.status.phase === "Failed") ?
+                                                    <Icon type="pf" name="error-circle-o" className="icon"/> : <Icon type="pf" name="ok" className="icon"/> }
+                                        {mobileClientBuild.kind} #{mobileClientBuild.metadata.annotations['openshift.io/build.number']} 
+                                        {(mobileClientBuild.status.phase === "Running") ?
+                                            " is running " : (mobileClientBuild.status.phase === "Failed") ? 
+                                            " failed " : " is complete "}
+                                </p>
+                                <p className="timestamp">
+                                            {this.convertTime(mobileClientBuild.status.startTimestamp)}
+                                            </p>
+                                </Col>
+                            </Row>
+                    </ListGroupItem>
+                ))}
+            </ListGroup>
+        </div> 
+        );
+    }
+    convertTime = (timeString) => {
```

Comment:
Can you check the resource I am pretty sure this exists on it.
https://github.com/aerogear/origin-web-console/blob/aerogear-mcp/app/views/overview/_mobile-client-row.html#L100?

Reply:
using react-moment to do this now.